src/components/profile-view/profile-view.jsx

```diff
@@ -0,0 +1,147 @@
+import React, { useState, useEffect } from "react";
+import { Button, Form, Row, Col, } from "react-bootstrap";
+import { MovieCard } from "../movie-card/movie-card";
+
+export const ProfileView = ({ movies }) => {
+    const storedToken = localStorage.getItem("token");
+    const [token] = useState(storedToken ? storedToken : null);
+    const [username, setUsername] = useState('');
+    const [password, setPassword] = useState('');
+    const [email, setEmail] = useState('');
+
+    const storedUser = localStorage.getItem("user");
+    const [user, setUser] = useState(storedUser ? storedUser : null);
```

Comment:
When I console.log user, I get only the username, meaning during login you only store the username instead of all the user data.
I could recommend that instead of just storing the username during login, you store the whole user data, then use the user id to query the user again, this time you can pick the fvaorite from it.